sources/packages/backend/apps/queue-consumers/src/processors/schedulers/sfas-integration/_tests_/sims-to-sfas-integration.scheduler.e2e-spec.ts

```diff
@@ -0,0 +1,249 @@
+import { DeepMocked } from "@golevelup/ts-jest";
+import MockDate from "mockdate";
+import * as faker from "faker";
+import { INestApplication } from "@nestjs/common";
+import { formatDate, QueueNames } from "@sims/utilities";
+import {
+  createTestingAppModule,
+  describeProcessorRootTest,
+  mockBullJob,
+} from "../../../../../test/helpers";
+import {
+  E2EDataSources,
+  createE2EDataSources,
+  createFakeCASSupplier,
+  createFakeUser,
+  saveFakeApplicationDisbursements,
+  saveFakeStudent,
+} from "@sims/test-utils";
+import * as Client from "ssh2-sftp-client";
+import { SIMSToSFASIntegrationScheduler } from "../sims-to-sfas-integration.scheduler";
+import { OfferingIntensity, Student, SupplierStatus } from "@sims/sims-db";
+import { getUploadedFile } from "@sims/test-utils/mocks";
+import { MoreThan } from "typeorm";
+import { addMilliSeconds, addYears } from "@sims/test-utils/utils";
+
+describe(describeProcessorRootTest(QueueNames.SIMSToSFASIntegration), () => {
+  let app: INestApplication;
+  let processor: SIMSToSFASIntegrationScheduler;
+  let db: E2EDataSources;
+  let sftpClientMock: DeepMocked<Client>;
+  let latestBridgeFileDate: Date;
+  let simsDataUpdatedDate: Date;
+  let mockedCurrentDate: Date;
+  const DATE_FORMAT = "YYYYMMDD";
+
+  beforeAll(async () => {
+    const { nestApplication, dataSource, sshClientMock } =
+      await createTestingAppModule();
+    app = nestApplication;
+    db = createE2EDataSources(dataSource);
+    sftpClientMock = sshClientMock;
+    // Processor under test.
+    processor = app.get(SIMSToSFASIntegrationScheduler);
+  });
+
+  beforeEach(async () => {
+    // Reset all SFAS bridge logs.
+    await db.sfasBridgeLog.delete({ id: MoreThan(0) });
```

Comment:
We also use truncate for these situations, as below:
https://github.com/bcgov/SIMS/blob/4e3b55ab77ffb49cf25f35eaa0fd0f8650e38e3f/sources/packages/backend/apps/queue-consumers/src/processors/schedulers/esdc-integration/student-loan-balances/_tests_/student-loan-balances-part-time-integration.scheduler.e2e-spec.ts#L64